jobs/build_codekit_triggers.groovy

```diff
@@ -1,29 +0,0 @@
-import util.Common
-Common.makeFolders(this)
-
-def j = job('sqre/infra/build-codekit-triggers') {
-  def repo = SEED_JOB.scm.userRemoteConfigs.get(0).getUrl()
-  def ref  = SEED_JOB.scm.getBranches().get(0).getName()
-
-  scm {
-    git {
-      remote {
-        github('lsst-sqre/sqre-codekit')
```

Comment:
I think Build Engineering needs to take over ownership of this.  SQuaRE isn't going to use it for anything.  We should then replace this trigger with a GHA.

Reply:
We have taken over this and have replaced it with GHA